source/_reference-g-s.html.md.erb

```diff
@@ -8346,6 +8348,17 @@ Web Content | **WEBCONTENT** | [WebContentWidgetContent](#webcontentwidgetconten
 **hyperlink** | [WidgetHyperlink](#widgethyperlink-object) | The widget has **when clicked** attribute set to that hyperlink (if present and non-null). Hyperlinks will have **interactionType**.
 **width** | integer | Original width of the image in pixels
 
+### ProfileField Object
+
+     |     |     |
+-----|-----|-----|
+**id** | number | Column Id
```

Comment:
This is to "Field Id", not the "Column Id"

Reply:
Simiarly, all of the "column" references here should be changed to "field" or "summary field"